src/_gettsim_tests/test_arbeitsl_geld_2.py

```diff
@@ -75,7 +75,7 @@
 ]
 
 
-YEARS = [2005, 2006, 2009, 2013, 2018, 2019, 2022, 2023]
+YEARS = [2005, 2006, 2009, 2013, 2018, 2019, 2022, 2024]
```

Comment:
Given how much this has changed at different points during the year, I believe we should be more fine-grained in the tests. 

I.e., instead of years, go for dates (or months) and have one test for January 2023 and another one for July 2023. 

Who knows what will change in 2024, so the above is not future-proof for sure.

Reply:
Sounds good. It shouldn't be too difficult to change the `year` column in the `.csv` to a `date` column which can contain entries like `2023-07`.